src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -3694,10 +3694,10 @@ public String configRewrite() {
    * @return Status code reply
    */
   @Override
-  public byte[] configSet(final byte[] parameter, final byte[] value) {
+  public String configSet(final byte[] parameter, final byte[] value) {
```

Comment:
Shouldn't we deprecate the original method first?

Reply:
@gkorland We are not removing the method. So I don't support deprecation. My idea is to add a warning message in jdoc. We can do that after we agree on this PR.

Reply:
I know we are not but we change the signature which will break both compile and runtime

Reply:
Are you suggesting to deprecate without an alternate?

Reply:
no, I suggest we add an new method and deprecate the old one

Reply:
@gkorland check #2525